packages/core/src/utils/helpers.css

```diff
@@ -9,6 +9,17 @@
   width: 1px;
 }
 
+.visually-hidden {
+  clip: rect(1px, 1px, 1px, 1px);
+  clip-path: inset(50%);
+  max-height: 1px;
+  max-width: 1px;
+  opacity: 0;
+  overflow: hidden;
+  position: absolute;
+  white-space: nowrap;
+}
+
```

Comment:
There is .hiddenFromScreen css above this. I guess that does the same as this code, but in a slightly different way. Combine these two or delete this one.

Looks like this has more tricks than the old one.

Reply:
Good points, I noticed it but didn't have courage to modify the old one. I will combine those and rename the old class to visually-hidden as I believe it's more commonly used. I try to make it without braking anything.

Reply:
I renamed hiddenFromScreen also on react side and pushed changes. Let me know if it's good or not.

Reply:
Actually renaming this is a breaking change requiring this to be pushed to major release, because this helpers.css is included in hds-core package. So maybe revert this for now and I can make a ticket about it 👍 A stable version 3 should be right around the corner and I think this core cookie consent will take quite some time to finish, so I don't think this will make it to 3.

Reply:
Or we might make the change ourselves to v.3 💡 We'll see!

Reply:
Good point! I will revert the changes from this PR